cypress/e2e/draw/draw-feat-circle.cy.ts

```diff
@@ -35,7 +35,10 @@ describe('Draw "Circle"', () => {
       )
       cy.dragVertexOnMap(200, 200, 300, 300)
       cy.get('*[data-cy="featItemLength"]').should('contain.text', '693.17 km')
-      cy.get('*[data-cy="featItemArea"]').should('contain.text', '38235.40 km²')
+      // test of 2 possible measurements due to precision issues between local tests and CI
```

Comment:
I think we can get rid or adapt this comment now :-) (I see you adapted it below)

Reply:
yes